project/dependencies.scala

```diff
@@ -36,7 +36,7 @@ object Versions {
   val junitInterfaceVersion: String = "0.13.3"
 
   val bspVersion = "2.1.0-M3"
-  val sbtStructureVersion: String = "2024.1.2"
+  val sbtStructureVersion: String = "2024.1.2+0-38de1c9e+20240519-1859-SNAPSHOT"
```

Comment:
I know it's invalid, but sbt structure release will be required before merge anyway